### DISCUSSION TITLE
How to change profile table which is used for authentication

### DISCUSSION THREAD

**Discussion by SAYANISHAN98** [supabase/supabase#28420] [General]
i already connected a table to authentication feature now i want to change that table but i don't know how to do it. If  can i use two tables for authentication function

**Comment by GaryAustin1** [comment]
You need to provide more info on what it is you are doing and trying/wanting to do.

**Reply by SAYANISHAN98** [reply]
i am trying to use the inbuilt authentication function. I am building a blood bank management system. So whenever i add new user to my donor table it should add the necessary data into authentication table which will be used by donors to login to the system . so now i want to know how can i do this
